<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Objects</string>
     </attribute>
     <widget class="QGroupBox" name="gb_obj">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>221</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Current Object</string>
      </property>
      <widget class="QLabel" name="lbl_object_txt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_obj_pos_x">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_obj_pos_y">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_obj_pos_z">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_obj_rot_x">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_obj_rot_z">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_obj_rot_y">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Rotation:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_obj_apply_physic">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>apply physics</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_obj_physic_static">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>static</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_obj_fixed">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>fixed</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_obj_visible">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>visible</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_object_shadow_only">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>shadow only</string>
       </property>
      </widget>
     </widget>
     <widget class="QListView" name="lv_objects_chooser">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>21</y>
        <width>161</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_objects_add">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QFrame" name="f_objects_selected">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_objects_active">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>221</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Lights</string>
     </attribute>
     <widget class="QGroupBox" name="gb_light">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Current light</string>
      </property>
      <widget class="QLabel" name="lbl_light_txt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_light_pos_x">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_light_pos_y">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_light_pos_z">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_light_rot_x">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_light_rot_z">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_light_rot_y">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Direction:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_light_r">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_light_g">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_light_b">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>G</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Intensity:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="lw_lights_active">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>New light</string>
      </property>
      <widget class="QComboBox" name="cb_lights_type">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentText">
        <string>directional</string>
       </property>
       <item>
        <property name="text">
         <string>directional</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>spot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>point</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pb_lights_add">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_lights_cast_shadow">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>cast shadow</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Scripts</string>
     </attribute>
     <widget class="QLabel" name="lbl_object_txt_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_scripts">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>201</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSceneEditor">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuSceneEditor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
